dashboard/test/factories.rb

```diff
@@ -42,7 +42,7 @@
 
   factory :user do
     birthday Date.new(1991, 3, 14)
-    sequence(:email) {|n| "testuser#{n}@example.com.xx"}
+    sequence(:email) {|n| "testuser#{n}_#{SecureRandom.hex}@example.com.xx"}
```

Comment:
Why is the sequence no longer enough to guarantee emails are unique?

FWIW we depend on GitHub not having collisions in a 20 byte digest.  `SecureRandom.hex` is 16 bytes which should be plenty long.